ivy/functional/backends/torch/activations.py

```diff
@@ -125,13 +125,16 @@ def log_softmax(
 @with_unsupported_dtypes(
     {
         "2.0.1 and below": (
-            "complex",
+            "bfloat16",
```

Comment:
This surprises me, torch is usually fine with `bfloat16` on cpu. Are you sure it's the function that doesn't support it, rather than the testing pipeline?